pkg/kudoctl/kudoinit/prereq/webhook.go

```diff
@@ -28,41 +29,43 @@ import (
 var _ kudoinit.Step = &KudoWebHook{}
 
 type KudoWebHook struct {
-	opts        kudoinit.Options
-	issuer      unstructured.Unstructured
-	certificate unstructured.Unstructured
+	opts kudoinit.Options
+
+	certManagerGroup      string
+	certManagerAPIVersion string
+
+	issuer      *unstructured.Unstructured
+	certificate *unstructured.Unstructured
 }
 
 const (
-	certManagerAPIVersion        = "v1alpha2"
-	certManagerControllerVersion = "v0.12.0"
+	certManagerOldGroup = "certmanager.k8s.io"
```

Comment:
What about a group -> api mapping instead like?
```
certManagerGroupAPI := map[string]string{
  "v1alpha1": "certmanager.k8s.io",
  "v1alpha2": "cert-manager.io",
}

```

Reply:
It's not necessarily a map... It might be a map of lists:
```
certManagerAPIs := map[string][]string{
  "certmanager.k8s.io": []string{ "v1alpha1"},
  "cert-manager.io": []string{ "v1alpha1", "v1alpha2", "v1alpha3" },
}
```
(at least I think "cert-manager.io" has "v1alpha1" as well in some version, it certainly has v1alpha2 and v1alpha3)

Reply:
So even easier to check all permutatations 👍